content/applications/inventory_and_mrp/manufacturing/shop_floor/shop_floor_overview.rst

```diff
@@ -205,3 +205,54 @@ name in the panel.
 .. image:: shop_floor_overview/operator-panel.png
    :align: center
    :alt: The operator panel of the Shop Floor module, showing three employees signed in.
+
+MO/WO prioritization
+====================
+
+The Shop Floor module uses the *scheduled date* entered on |MOs| to prioritize the |MOs| and work
```

Comment:
totally a suggestion since we bold app names but shop floor isn't an app. Not really sure what to do here haha
```suggestion
The **Shop Floor** module uses the *scheduled date* entered on |MOs| to prioritize the |MOs| and work
```